pages/templates/pages/group_index_page.html

```diff
@@ -69,6 +69,18 @@ <h4 class="mb20 mt0">No group in your area?</h4>
   </div>
 </div>
 
+<div class="container pt4 pb2">
+  <div class="row">
+    <div class="col-xs-12">
+      <h3>
+        In addition to Local Groups, you can also take action on national
+        Our Revolution campaigns.
```

Comment:
@cjmabry thoughts about content here? design too?

Reply:
This is fine for now. We'll redo when we do the public groups portal landing page redesign.